tests/unit/test_refine.py

```diff
@@ -159,7 +159,7 @@ def test_refine(
 
         When ``use_cached_row_sum_mean=True``, we expect the corresponding cached value
         in the coreset object to be used by refine, otherwise, we expect the kernel's
-        calculate_kernel_matrix_row_sum_mean to be called (exactly once).
```

Comment:
Maybe refer to use_cached_row_mean rather than use_cached_row_sum_mean now?

Reply:
Agreed